code/game/objects/items/ego_weapons/special.dm

```diff
@@ -96,3 +96,20 @@
 	icon_state = "freshman"
 	damtype = PALE_DAMAGE
 	armortype = PALE_DAMAGE
+
+/obj/item/ego_weapon/wbaton
+	name = "W corp baton"
+	desc = "A glowing blue baton used by W corp."
+	icon_state = "wbatong"
+	inhand_icon_state = "wbatong"
+	force = 45
```

Comment:
That's a pretty strong hit without any special requirements.

Reply:
Been overhauled entirely, check the new one